app/uk/gov/hmrc/thirdpartydeveloperfrontend/domain/models/connectors/DeskproHorizonTicket.scala

```diff
@@ -27,14 +27,20 @@ case class DeskproHorizonTicketPerson(
 
 case class DeskproHorizonTicketMessage(
     message: String,
-    format: String
+    format: String = "html"
   )
 
+object DeskproHorizonTicketMessage {
+  def fromRaw(message: String): DeskproHorizonTicketMessage = DeskproHorizonTicketMessage(message.replaceAll(Properties.lineSeparator, "<br>"))
+
+}
+
 case class DeskproHorizonTicket(
     person: DeskproHorizonTicketPerson,
     subject: String,
     message: DeskproHorizonTicketMessage,
-    brand: Int
+    brand: Int = 3,
```

Comment:
I'm guessing this number could be different in QA and Prod, as it's an ID. Could it be a config instead?